src/webapp/components/app-bar/GlassAppBar.tsx

```diff
@@ -10,6 +10,7 @@ import glassLogo from "../../assets/glass-logo.png";
 import { Box, MenuItem, Select } from "@material-ui/core";
 import styled from "styled-components";
 import i18n from "@eyeseetea/d2-ui-components/locales";
+import { useTestContext } from "../../contexts/test-context";
```

Comment:
@xell12686  : Is this test context required or should it be removed?

Reply:
hi @9sneha-n , you may remove it you want to replace it with something else. I might be renaming it with a countryContext depending on how the next tasks come about.